components/WorkoutList/WorkoutList.js

```diff
@@ -44,10 +45,14 @@ export default function WorkoutList() {
           </Fragment>
           ))}
         </ListContainer>
-        <Buttons>
+          <StartButton href={`workouts/${workout._id}`}>
+            <FontAwesomeIcon icon={faPlay} />
+            <p>Start</p>
+          </StartButton>
+        {/* <Buttons>
 
           <DeleteButton onClick={() => handleDelete(workout._id)}><FontAwesomeIcon icon={faTrash} /></DeleteButton>
-        </Buttons>
+        </Buttons> */}
```

Comment:
Instead of putting it inside a comment, I would remove code that is not needed entirely.